### ISSUE TITLE
Increase click targets on navigator wheel

### ISSUE THREAD

**Issue by alicerunsonfedora** [alicerunsonfedora/mcmaps#23]
On iOS and iPadOS, the touch targets for the arrow buttons in the navigator wheel are increased to allow fingers to more easily tap the buttons. On macOS, it is currently set to be much smaller, given the precise nature of the cursor/pointer.

However, in initial testing, it seems the area is a bit too small and requires more precision than what’s required. These targets should be increased or adjusted to allow leniency.
